charts/registry-scanner/values.yaml

```diff
@@ -113,6 +113,8 @@ config:
       temporaryVolumeSizeLimit: 2Gi
     # Force the scan to happen on the client component rather than relying on backend scanning
     disablePlatformScanning: false
+    # NodeSelector for child jobs. If only .Values.nodeSelector is specified, child jobs will inherit the same nodeSelector as the main pod
+    # inlineJobNodeSelector: {}
```

Comment:
let's rename this to nodeSelector and move it under jobs (line 99), that's where we are putting the job configs 🙏 
does it make sense to you?
so it will be
```
jobs:
     nodeSelector: {}
     ttlSecondsAfterFinished: 600

```